nrf_wifi/fw_if/umac_if/src/tx.c

```diff
@@ -1159,6 +1450,44 @@ static void tx_done_tasklet_fn(unsigned long data)
 }
 #endif /* CONFIG_NRF700X_TX_DONE_WQ_ENABLED */
 
+#ifdef CONFIG_NRF700X_RAW_DATA_TX
+enum nrf_wifi_status (nrf_wifi_fmac_rawtx_done_event_process)(
```

Comment:
Why the brackets for function name?

Reply:
just copied the format from nrf_wifi_fmac_tx_done_event process. can remove the brackets

Reply:
done